src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -170,15 +171,32 @@ const ChatsList: React.FunctionComponent = () => {
                     setSelectedUserProfile={noopHandler}
                   />
                   <div className="private-message-content">
-                    <div className="private-message-author">{profileName}</div>
-                    <div className="private-message-last">
+                    <div
+                      className={`private-message-author ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
+                      {profileName}
+                    </div>
+                    <div
+                      className={`private-message-last ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
                       {lastMessageExchanged.text}
                     </div>
                   </div>
-                  {lastMessageExchanged.from !== user?.uid &&
-                    !lastMessageExchanged.isRead && (
-                      <div className="not-read-indicator" />
-                    )}
+                  {lastMessageExchanged.from !== user?.uid && (
+                    <div
+                      className={`private-message-time ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
```

Comment:
You've removed `not-read-indicator` class here, seemingly in favour of `unread`, yet in `src/components/molecules/ChatsList/ChatsList.scss` you set a style for `not-read-indicator`.

Is this correct?

Reply:
It needs to be removed, nice catch.